ibft/ibft_test.go

```diff
@@ -32,20 +41,10 @@ func TestIBFT(t *testing.T) {
 
 		ibft := New(replay.Storage, me, replay.Network, params)
 		instances = append(instances, ibft)
-
-		//instances = append(instances, NewInstance(InstanceOptions{
-		//	Logger:    logger,
-		//	Me:        me,
-		//	Network:   replay.Network,
-		//	Consensus: weekday.New(),
-		//	Params:    params,
-		//}))
-		//instances[i].StartEventLoop()
-		//instances[i].StartMessagePipeline()
 	}
 
 	// start repeated timer
-	ticker := time.NewTicker(10 * time.Second)
+	ticker := time.NewTicker(20 * time.Second)
```

Comment:
minor for learning purposes: why to double the timer?

Reply:
to see failures of consensus